src/scss/2022/elements/_headings.scss

```diff
@@ -1,22 +1,32 @@
 @use "../tools/typography";
 
+h1,h2,h3,h4,h5,h6 {
+  font: inherit;
+  margin: unset;
+}
+
 h2:not([class]) {
   @include typography.style('heading-1');
+  margin: 1em 0;
```

Comment:
More of a question ~ would we want to create a variable for this margin since it has been duplicated a few times?

Reply:
That is absolutely a fair question and in most other cases, I would say yes. But this resetting defaults for the main content elements. It is something that will only be done in this file and this pattern is highly unlikely to be used further. When using the typography.style mixin in other files, the margin is either removed or uses something fro the spacing system which is done in `rem` values. The `em` value on this page is a good default for content elements since it adds spacing above and below the element that is equal to the font-size. If that doesn’t make sense, let’s chat!